public/xmlns/tract.xsd

```diff
@@ -219,6 +219,11 @@
                         </xs:documentation>
                     </xs:annotation>
                 </xs:attribute>
+                <xs:attribute name="button-color" type="content:colorValue" use="optional">
```

Comment:
how about `control-color` we already use `control` elsewhere for similar functionality. see: https://github.com/CruGlobal/mobile-content-api/blob/master/public/xmlns/manifest.xsd#L53